Gems/Multiplayer/Code/Source/AutoGen/AutoComponent_Source.jinja

```diff
@@ -318,7 +318,11 @@ void {{ ClassName }}::{{ UpperFirst(Property.attrib['Name']) }}({{ ', '.join(par
     constexpr AzNetworking::ReliabilityType isReliable = Multiplayer::ReliabilityType::Unreliable;
 {% endif %}
 
+{%     if (InvokeFrom == 'Server' or InvokeFrom =="Client") and HandleOn == 'Authority' %}
```

Comment:
I'd consider removing the 'and HandleOn == 'Authority'' bit, since if I make an InvokeFrom=='Server' and HandleOn=='Autonomous', this will cause the autogen to fall into the controller case when it's actually being invoked from a component. Other than that I think you've nailed the change.